First/src/firstPackage/EqualOccurance.java

```diff
@@ -0,0 +1,27 @@
+package firstPackage;
+
+public class EqualOccurance {
+    public boolean areOccurrencesEqual(String inputString) {
+        int size = inputString.length();
+        int[] counts = new int[size];
+        int max = 0;
+        for (char inputChar : inputString.toCharArray()) {
+            counts[inputChar - 'a']++;
+            max = Math.max(max, counts[inputChar - 'a']);
+        }
+        for (int count : counts) {
+            if (count != 0 && count != max) {
+                return false;
+            }
```

Comment:
use different logic

Reply:
logic is not proper, please check it again

Reply:
Done! Updated code of Assignment no.1.